integration-commons/src/main/java/org/symphonyoss/integration/exception/ExceptionMessageFormatter.java

```diff
@@ -100,7 +117,27 @@ public static String format(String component, String message, Throwable t) {
    *
    * Component: <Component>
    * Message: <Message>
-   * Root cause: <Root cause>
+   * Solutions:
+   * <Solution A>
+   * Stacktrace: <Trace>
+   * @param component The component where the exception was thrown.
+   * @param message The message why the exceptions happened.
+   * @param solution The solution provided by exceptions.
+   * @param t The exception caused the problem.
+   * @return Formatted message
+   */
+  public static String format(String component, String message, String solution,
+      Throwable t) {
+    List<String> solutions = new ArrayList<>();
+    solutions.add(solution);
+    return getMessage(component, message, solutions, t);
```

Comment:
Could we use varargs parameters insteadof a list of strings? 

Something like that: getMessage(String component, String message, Throwable t, String... solutions)